tests/Unit/Core/Grid/Action/Row/AccessibilityChecker/DeleteCartAccessibilityCheckerTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Tests\Core\Grid\Action\Row\AccessibilityChecker;
+
+use PHPUnit\Framework\TestCase;
+use PrestaShop\PrestaShop\Core\Grid\Action\Row\AccessibilityChecker\DeleteCartAccessibilityChecker;
+
+class DeleteCartAccessibilityCheckerTest extends TestCase
+{
+    private const CART_RECORD_STATUS_NOT_PLACED = [
+        'id_cart' => '15',
+        'date_add' => '2020-01-31 03:20:54',
+        'status' => 'Not placed',
+        'id_order' => null,
+        'customer_name' => 'J. DOE',
+        'carrier_name' => '--',
+        'id_guest' => '0',
+        'online' => 'No',
+        'cart_total' => '$0.00',
+        'is_order_placed' => false,
+        'shop_name' => 'testadas',
+    ];
+
+    private const CART_RECORD_STATUS_ORDER_ID = [
+        'id_cart' => '11',
+        'date_add' => '2020-01-22 10:21:43',
```

Comment:
Value around `status` are useless here, only adding confusion.
Please test with 
```
-10
2.1
2
6463165813
not
not placed
12notready
-12zef210
-12.02e1f
```

Reply:
and with a non existing status key

Reply:
After the tests run; return !is_numeric($record['status']); still gives better results :thinking: